src/protect-api/README.md

```diff
@@ -123,10 +148,53 @@ Later the web app can make calls to protected Apis in the name of the signed-in
 
    :book: Since the request is sent without a Bearer Token, it is expected to receive an Unauthorized response `401`. The web API is now protected
```

Comment:
Nit

```suggestion
   :information_source: Since the request is sent without a Bearer Token, it is expected to receive an Unauthorized response `401`. The web API is now protected
```